### ISSUE TITLE
Can't build mungefs as a non-external

### ISSUE THREAD

**Issue by alanking** [jasoncoposky/mungefs#10]
Attempting to build mungefs with iRODS externals as shipped and installed in the appropriate location (`/opt/irods-externals`) no longer works. Needed headers cannot be located due to changes in this commit: 2d7a4f6ff56e64b560f582a21438162a1e8f36ab

Not a blocker, but could be a hindrance for people trying to build manually.

**Comment by trel** [comment]
Detect the presence of some of the CMake variables... and assign some default values if they are not present.

for example... for each of AVRO_PATH, BOOST_PATH, ZMQ_PATH, CPPZMQ_PATH...


```
if (NOT AVRO_PATH)
  set(AVRO_PATH "/really/excellent/default/path" CACHE STRING "The default AVRO_PATH" FORCE)
  message(STATUS "Setting unspecified AVRO_PATH to '${AVRO_PATH}'. This is the correct setting for normal builds.")
endif()
```

**Comment by SwooshyCueb** [comment]
Fixed by #32
